JEP.md

```diff
@@ -178,6 +178,46 @@ HTTP PUT /api/telemetry/event
 
 1. Is this work done on the standalone jupyter-server implementation or on the classic jupyter/notebook?
 
+### JupyterHub
+
+JupyterHub would use the same underlying python library as the notebook server, and be configured
+in the same way.
+
+JupyterHub relies on extension through Spawners, Authenticators, Proxies & Services to
```

Comment:
nit: Can you link these to the documentation for folks that need more info

Reply:
Done!